Manual.md

```diff
@@ -192,6 +194,22 @@ via `make install` or any other compatible method.
 successfully, the phase will be skipped later (unless its work directory
 `${wrksrc}` is removed with `xbps-src clean`).
 
+<a id="commonbuildproblems"></a>
+### Common Build Problems
+
+<a id="unresolvableshlib"></a>
+#### Unresolvable shlib
+
+If you get an error like the following while building:
+```
+enchant2-2.2.15_4: broken, unresolvable shlib `libicudata.so.69'
+```
+
+It's likely that the Void Buildbot has recently rebuilt a library and has not rebuilt some packages that depend on it.
+You can check https://build.voidlinux.org/ for the current status of the builders.
+
+Once the dependent packages have finished building on Buildbot your build should succeed.
```

Comment:
That said, I'm not sure how much this fits in Manual. If anything, it belongs more in the README, since that's about using `xbps-src` rather than about writing templates.

And even so, it's already documented in https://docs.voidlinux.org/xbps/troubleshooting/common-issues.html#shlib-errors

Maybe we should simply add a link to that, at most?

Reply:
Fair enough, I wasn't really sure where to put it, and wasn't aware of that page.

IMO it's still helpful to explain that this can happen when using `xbps-src` and not just `xbps-install`

Reply:
What do you think about putting it in the README and linking to the docs page, then?

Reply:
Its been done as of d1de86e07eaa8c8fdde03bf25c9449312a6f95f9